### ISSUE TITLE
Mentor's persona: a mentor's profile page mock-up design

### ISSUE THREAD

**Issue by ipsha21** [ipsha21/altruism-in-tech#13]
Coming up with a basic layout of a mentor's profile page, including all the features discussed in #12, which would eventually be transformed into code.

**Comment by ipsha21** [comment]
#12

**Comment by pransh15** [comment]
I really like the [Mozillians'](https://www.mozillians.org) profile pages. Can we go ahead with something like that with a few extra buttons for Time Slots and Asking questions?

**Comment by pransh15** [comment]
![mentor s profile_mockup](https://cloud.githubusercontent.com/assets/18158596/26282842/f1030c88-3e36-11e7-93f3-63fed99b7f72.jpg)

The Mentor's Profile Mock Up. Looks good?

**Comment by amirad** [comment]
That looks good. Can we have a section for mentors that say "I can help others with....." It would be cool to be even more direct with things mentors can help with and vise versa, areas where mentees need support.

Ex. I can help with:
•proposal writing
•public speaking
•telling your story
*event planning

**Comment by ipsha21** [comment]
Hey @amirad, I really like this idea. Would you suggest to have check-boxes for this in the mentor's registration form (with options like these and others), or a simple input text field for mentors to fill in?

**Comment by amirad** [comment]
Ohhhhhh I was thinking about an open text field but now that you mention it, check boxes would be way easier. Maybe when mentors sign up they check all the ways they can help and that is displayed on their profile. 

You can even run on-going surveys to mentees to ask what they need help in and add it to the list so mentors can identify if they have that skill. Oh the possibilities. I'm getting excited!

**Comment by ipsha21** [comment]
Yes, that would be incredibly convenient for the mentors. Can also add an "others" option with a text field for them to add something they feel is not one of the given options. 

+1 for conducting surveys, would be really helpful! :D

**Comment by pransh15** [comment]
![mentor s profile_mockup2](https://cloud.githubusercontent.com/assets/18158596/26723928/79679158-47b4-11e7-8546-08e502489012.jpg)

How does this look?

**Comment by ipsha21** [comment]
@pransh15 The design looks good, we could add a "write a testimonial" button at the end of the page. :)
Thanks for this mock up! 🥂

**Comment by pransh15** [comment]
![mentor s profile_mockup3](https://cloud.githubusercontent.com/assets/18158596/26727348/2ad90f7a-47c4-11e7-9a64-076a38699a10.jpg)

I hope this looks fine. 🥂

**Comment by amirad** [comment]
LOOKS FINE!!!!!
